File: cakk-external/src/main/java/com/cakk/external/vo/CertificationMessage.java
package com.cakk.external.vo;

public record CertificationMessage(
	String businessRegistrationImageUrl,
	String idCardImageUrl,
	String emergencyContact,
	String message,
	Long userId,
	String userEmail,
	String shopName,
	Double latitude,
	Double longitude
) {
}

File: cakk-external/src/main/kotlin/com/cakk/external/vo/message/CertificationMessage.kt
package com.cakk.external.vo.message

data class CertificationMessage(
    val businessRegistrationImageUrl: String,
    val idCardImageUrl: String,
    val emergencyContact: String,
    val message: String,
    val userId: Long,
    val userEmail: String,
    val shopName: String,
    val latitude: Double,
    val longitude: Double
)
